tests/func/test_analytics.py

```diff
@@ -14,7 +14,8 @@ def test_daemon_analytics(mock_send, tmp_path):
 
 @mock.patch("dvc.analytics.collect_and_send_report")
 @mock.patch("dvc.analytics.is_enabled", return_value=True)
-def test_main_analytics(mock_is_enabled, mock_report, dvc_repo):
+def test_main_analytics(mock_is_enabled, mock_report, tmp_dir, dvc):
+    tmp_dir.gen("foo")
```

Comment:
I would say you want empty `foo` explicitly.

Reply:
It is more that I don't care about the content of `foo` :sweat_smile: